atd-vze/.env

```diff
@@ -5,3 +5,4 @@ REACT_APP_AUTH0_CLIENT_ID=2qbqz2sf8L9idBOwn0d5YA9efNgQbL7c
 REACT_APP_MAPBOX_TOKEN="pk.eyJ1Ijoiam9obmNsYXJ5IiwiYSI6ImNqemU4OGQzaTBlZjczYm1rZWg1c2lidTcifQ.2eooq4Q7dv_1Le5la9D96Q"
 REACT_APP_NEARMAP_KEY="YzI5YTM1ZWMtMWRkNy00NGU4LTkxMWQtNDI3ZDcxNjVjMzg3"
 REACT_APP_CR3_API_DOMAIN="https://atd-vz-api-staging.austinmobility.io"
+REACT_APP_ENV="local"
```

Comment:
new env variables! john and i decided to just put `local` here even though when you spin up the local stack you could point to staging data depending on what `REACT_APP_HASURA_ENDPOINT` is commented out. maybe we want to think about doing away with the staging option here, but i do find it helpful for now when i just have a front end code change that i want to check without pulling a new copy of prod data and starting up the local db which can take some time.